torch/csrc/jit/passes/onnx/constant_fold.cpp

```diff
@@ -377,10 +428,11 @@ c10::optional<at::Tensor> runTorchBackendForOnnx(
     return c10::optional<at::Tensor>(updated_val);
   } else if (node->kind() == onnx::Gather) {
     assert(inputTensorValues.size() == 2);
-    if (!node->hasAttributeS("axis")) {
-      return c10::nullopt;
+    // default axis = 0
```

Comment:
is this valid change for all opsets?

Reply:
For all opsets, the specs on Gather axis are the same: `axis : int (default is 0)`. And the CI build all passed with this change.